mkDub.nix

```diff
@@ -98,7 +100,7 @@ in {
 
       export HOME=$PWD
       ${lib.concatMapStringsSep "\n" dub-add-local deps}
-      dub build -b release --combined --skip-registry=all
+      dub build -b  ${buildType} --combined --skip-registry=all ${extraDubFlags}
```

Comment:
There's an extraneous space after `-b`. Also wondering whether the extra space at then end (after `=all`) causes a bunch of existing hashes to change, in the `extraDubFlags==""` case?

Reply:
It may be, I don't know either. I guess the way to add it would be to add a special case when both `buildType` and `extraDubFlags` have their default values but that'd be ugly. Worth it?

Reply:
You're probably right